bioio_base/standard_metadata.py

```diff
@@ -1,5 +1,28 @@
+import logging
 from dataclasses import dataclass
+from datetime import datetime
 from typing import Optional, Sequence
+from zoneinfo import ZoneInfo
+
+from lxml.etree import XML, _Element
+from ome_types import OME
+
+# XML namespace for OME
+OME_NS = {"": "http://www.openmicroscopy.org/Schemas/OME/2016-06"}
+
+log = logging.getLogger(__name__)
+
+# Valid objectives for comparison when parsing Objective
+VALID_OBJECTIVES = [
+    "63x/1.2W",
+    "20x/0.8",
+    "40x/1.2W",
+    "100x/1.25W",
+    "100x/1.46Oil",
+    "44.83x/1.0W",
+    "5x/0.12",
+    "10x/0.45",
```

Comment:
where did these come from?

Reply:
This is just ported over from https://github.com/bioio-devs/bioio-czi/pull/43 as for where it came before that this is the breakdown we have from MXS I believe. We probably dont need it to be honest.

Reply:
I was wondering if it is too AICS-specific to be here.

Reply:
It seems these are only used to validate the objectives - worst case we just ignore the objective. What is the alternative you are thinking - avoid parsing?

Reply:
yeah why add any formatting here

Reply:
I really don't know where the list came from, it just seems like a list of known lenses that we have used at AICS but I don't know if it's something more global that would work for anyone, or if we would need to modify it in the future to keep it up to date.

Reply:
Part of what I was thinking was why validate against any list?

Reply:
removed in [5300d36](https://github.com/bioio-devs/bioio-base/pull/38/commits/5300d36a807a4541067370815c8432bcf8c71d53). This does mean that the formatting of the metadata value will be different. This also got me wondering why we format objective as `{magnification}x/{lens_na}{immersion}` why not just provide all of those individually.